server/src/storage/uncompressed_data_manager.rs

```diff
@@ -646,23 +698,29 @@ mod tests {
         data_manager.spill_finished_buffer().await;
 
         assert!(remaining_memory < data_manager.uncompressed_remaining_memory_in_bytes);
+        assert_eq!(data_manager.used_uncompressed_memory_log.values.len(), 2);
+
+        // The used disk space log should have an entry for when the uncompressed data manager is
```

Comment:
Maybe add similar comments for other tests were it is not obvious how many and which entries should be in the `Log`s.

Reply:
I think most of the places were clear but I have added a few comments to explain why the used disk space log has one more entry than might be expected.